pkg/app/pipedv1/plugin/kubernetes/deployment/server.go

```diff
@@ -60,21 +60,21 @@ type logPersister interface {
 type DeploymentService struct {
 	deployment.UnimplementedDeploymentServiceServer
 
-	RegexPool    *regexpool.Pool
-	Logger       *zap.Logger
-	ToolRegistry toolRegistry
-	Loader       loader
-	LogPersister logPersister
+	logger       *zap.Logger
+	loader       loader
+	logPersister logPersister
 }
 
 // NewDeploymentService creates a new planService.
 func NewDeploymentService(
 	logger *zap.Logger,
+	toolRegistry toolRegistry,
+	logPersister logPersister,
 ) *DeploymentService {
 	return &DeploymentService{
-		RegexPool:    regexpool.DefaultPool(),
-		Logger:       logger.Named("planner"),
-		ToolRegistry: nil, // TODO: set the tool registry
+		logger:       logger.Named("planner"),
+		loader:       provider.NewLoader(toolregistry.NewRegistry(toolRegistry)),
```

Comment:
IMO: How about changing the interface name of `toolRegistry` to `toolClient`?
At first, I was confused that it created a new tool Registry with tool registry.

Reply:
I agree that it's confusing. I'll rename it as you suggested. Thanks.

Reply:
@ffjlabo 
fixed on https://github.com/pipe-cd/pipecd/pull/5381/commits/ee332f54b0720e0612c4b98cc1f8c31fbbb548ae
Thanks.